main.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+import "purema-cli/cmd"
```

Comment:
Isn't a convention to use the remote repository in the path? I mean like this: `"github/LautaroNavarro/purema-cli/cmd"`

Reply:
I don't know, I think that go first try to take it locally, it means that will search for this in the $GO_PATH env variable, and then will try to get it from GitHub
Anyway trying to get the cmd from GitHub remotely will fail since it does not exists